src/test/java/org/itsallcode/openfasttrace/ITestSelfTrace.java

```diff
@@ -0,0 +1,71 @@
+package org.itsallcode.openfasttrace;
+
+/*-
+ * #%L
+ * OpenFastTrace
+ * %%
+ * Copyright (C) 2016 - 2018 itsallcode.org
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public
+ * License along with this program.  If not, see
+ * <http://www.gnu.org/licenses/gpl-3.0.html>.
+ * #L%
+ */
+import static org.hamcrest.Matchers.equalTo;
+import static org.junit.Assert.assertThat;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+
+import org.itsallcode.openfasttrace.core.*;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ITestSelfTrace
+{
+    private Oft oft;
+
+    @Before
+    public void setUp()
+    {
+        this.oft = new OftRunner();
+    }
+
+    @Test
+    public void testSelfTrace()
```

Comment:
:+1: * 100
That is a very elegant way to create a build breaker. I only came up with an additional build step but this is beautiful.